pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -35,15 +35,23 @@ def test_min_max(self):
         assert _min == "d"
         assert _max == "a"
 
-    def test_min_max_empty(self):
+    @pytest.mark.parametrize(
```

Comment:
can you move the ordered=True tests above to a new test and name appropriately and change the name of your test to be similar (e.g. include _ordered in the name)